x11-themes/vanilla-dmz-xcursors/vanilla-dmz-xcursors-0.4.5.ebuild

```diff
@@ -0,0 +1,51 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+MY_PN="Vanilla-DMZ"
+DESCRIPTION="Style neutral scalable cursor theme (white version)"
+HOMEPAGE="http://jimmac.musichall.cz/"
+SRC_URI="mirror://debian/pool/main/d/dmz-cursor-theme/dmz-cursor-theme_${PV}.tar.xz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~arm64 ~ppc ~ppc64 ~sparc ~x86"
+IUSE=""
+
+DEPEND="x11-apps/xcursorgen"
+
+S="${WORKDIR}/dmz-cursor-theme-${PV}"
+
+src_compile() {
+	cd "${S}/DMZ-White/pngs" && ./make.sh
+}
+
+src_install() {
+	insinto /usr/share/icons/Vanilla-DMZ/cursors
+	doins "${S}/DMZ-White/xcursors/"* || die "doins failed"
+	insinto /usr/share/icons/Vanilla-DMZ
+	doins "${S}/DMZ-White/index.theme" || die "doins failed"
+}
+
+pkg_postinst() {
+	elog "To use this set of cursors, edit or create the file ~/.Xdefaults"
```

Comment:
Can't this cursor theme be selected via normal DE settings?

Reply:
> Can't this cursor theme be selected via normal DE settings?
This stuff is in the 0.4 ebuild, which is the only reason I kept it in. I'll remove it.